ouroboros-network/sim-tests-lib/Test/Ouroboros/Network/Testnet.hs

```diff
@@ -105,7 +105,6 @@ tests =
     , testProperty "diffusionScript command script valid"
                    prop_diffusionScript_commandScript_valid
     ]
-#if !defined(mingw32_HOST_OS)
```

Comment:
Wont this cause problems for a developer attempting to run the test suite on windows?

Reply:
I hope not, I haven't checked. But otherwise a Windows dev doesn't have a way to run the tests locally.  If it breaks it's better to know it & fix this.